spatial_image_multiscale.py

```diff
@@ -313,6 +314,140 @@ def align_chunks(current_input, dim_factors):
                 name=image.name, promote_attrs=True
             )
             current_input = downscaled
+    elif method is Methods.ITK_GAUSSIAN:
+        import math
+        import itk
+        assert 'VkFFTBackend' in dir(itk)  # Verify itk-vkfft installed
+
+        # output spacing is input_spacing * shrink_factors
+
+        def compute_sigma(input_spacings, shrink_factors) -> list:
+            assert len(input_spacings) == len(shrink_factors)
+            output_spacings = [input_spacing * shrink for input_spacing, shrink in zip(input_spacings, shrink_factors)]
```

Comment:
Can we put a reference to the logic here in a comment?

Reply:
Added